test/dlc_tests/eks/tensorflow/inference/test_eks_tensorflow_inference.py

```diff
@@ -9,6 +9,55 @@
 import test.test_utils as test_utils
 
 
+@pytest.mark.model("mnist")
+def test_eks_tensorflow_neuron_inference(tensorflow_inference):
```

Comment:
can you use `tenserflow_inference_neuron` to avoid adding all the pytest.skips?

Reply:
tenserflow_inference_neuron cant be used because that calls the docker run etc and this one deals with k8s service deployment